src/js/common/schemaform/save-in-progress/reducers.js

```diff
@@ -68,7 +68,10 @@ export const saveInProgressReducers = {
 
     // if we’re prefilling, we want to use whatever initial data the form has
     if (state.prefillStatus === PREFILL_STATUSES.pending) {
-      const formData = _.merge(state.data, action.data.formData);
+      let formData = _.merge(state.data, action.data.formData);
+      if (state.storePrefillStatus) {
+        formData = _.merge(state.data, action.data.formData, { prefilled: true });
+      }
```

Comment:
`prefillStatus` will be stored to ensure that the applicant is presented with a consistent experience if they save their form and log in again later to complete it.